docs/contribute/code.md

```diff
@@ -0,0 +1,7 @@
+# Contributing Code
+
```

Comment:
Do we want to add an issue for each feature and like it to the pr?

Reply:
i don't know. should we? doesn't seem too important